### PR TITLE
Bug&Feat/Bidding page re-render into input, along with other front-end rendering

### PR DESCRIPTION
**Pull request by alisonzthu**

### CODE REVIEW

File: client/src/components/Auction.jsx [hunk 1]
```diff
@@ -40,7 +37,7 @@ class Auction extends Component {
       dispatch(Auctions.fetchingAnAuction(false));
       dispatch(Auctions.fetchAuctionErrored(true, err));
     });
-    //lala:
+
     if (user.username) {
```
Comment (abianco3) [inline client/src/components/Auction.jsx]: I think we should try and be consistent with using ES6, especially now that we are closer to the end and need to polish up our codebase.

File: server/controllers/auctions.js [hunk 2]
```diff
@@ -74,14 +74,14 @@ module.exports = function(io) {
...
-    // console.log('bidder', bid.bidder_id);
     // console.log(io.sockets.sockets, 'listof sockets in auctions handler');
+    console.log('bid: ', bid);
     model.createBid(bid)
     .then((bid) => {
       const update = {};
       update.auction_id = req.params.auctionId;
       update.bid = bid;
-
+      console.log('update: ', update);
```
Comment (abianco3) [inline server/controllers/auctions.js]: remove please

File: client/src/components/Auction.jsx (hunk 1 above)
Comment (alisonzthu) [inline client/src/components/Auction.jsx]: Agreed!! Will try to have less of these in future pull requests.....

Comment (alisonzthu) [inline client/src/components/Auction.jsx]: Agreed!! So which line of code do you think I could improve? I'm a bit confused.....

**Pull request merged**